engine/Shopware/Models/Order/Document/Type.php

```diff
@@ -105,6 +105,13 @@ class Type extends ModelEntity
      */
     private $pageBreak;
 
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="key", type="string", nullable=false)
```

Comment:
We should move this (and all other changes in this file) to https://github.com/shopware/shopware/pull/1464.